kuma/health/views.py

```diff
@@ -156,3 +161,39 @@ def status(request):
     data['services']['test_accounts'] = test_account_data
 
     return JsonResponse(data)
+
+
+@csrf_exempt
+@require_POST
+def csp_violation_capture(request):
+    """
+    Capture CSP violation reports, forward to Sentry.
+
+    HT @glogiotatidis https://github.com/mozmeao/lumbergh/pull/180
+    HT @pmac, @jgmize https://github.com/mozilla/bedrock/pull/4335
+    """
+    if not settings.CSP_REPORT_ENABLE:
+        # mitigation option for a flood of violation reports
+        return HttpResponse()
+
+    data = client.get_data_from_request(request)
+    data.update({
+        'level': logging.INFO,
+        'logger': 'CSP',
+    })
+    try:
+        csp_data = json.loads(request.body)
+    except ValueError:
+        # Cannot decode CSP violation data, ignore
+        return HttpResponseBadRequest('Invalid CSP Report')
+
+    try:
+        blocked_uri = csp_data['csp-report']['blocked-uri']
+    except KeyError:
+        # Incomplete CSP report
+        return HttpResponseBadRequest('Incomplete CSP Report')
+
+    client.captureMessage(message='CSP Violation: {}'.format(blocked_uri),
```

Comment:
I feel we do, but it could be better. We don't have this enabled in production for fear of DDOSing ourselves. Mostly we see blocked things being injected by addons.